Source/WebKit/UIProcess/API/Cocoa/WKWebsiteDataStore.mm

```diff
@@ -299,6 +299,33 @@ + (void)_deleteDefaultDataStoreForTesting
     return WebKit::WebsiteDataStore::deleteDefaultDataStoreForTesting();
 }
 
++ (void)_fetchAllIdentifiers:(void(^)(NSArray<NSUUID *> *))completionHandler
+{
+    auto completionHandlerCopy = makeBlockPtr(completionHandler);
+    WebKit::WebsiteDataStore::fetchAllDataStoreIdentifiers([completionHandlerCopy](auto&& identifiers) {
+        auto result = adoptNS([[NSMutableArray alloc] initWithCapacity:identifiers.size()]);
+        for (auto identifier : identifiers) {
+            if (auto uuid = identifier.createNSUUID())
```

Comment:
When does create return nil? Why would skipping in that case be OK?

Reply:
This should not return nil, since fetchAllDataStoreIdentifiers returns valid UUID, so I will just remove this.